### ISSUE TITLE
Add frameworks to project pages

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/website#460]
### Overview
Project's need a way to communicate Frameworks to volunteers

### Action Items

- [x] Add a field for Frameworks to the project.md card wiki
- [x] Write the logic for including this in the display.  It should appear under Languages.
- [x] Use team framework info collected on  - Issue #459 
- [x] Add to all project cards as one commit (list projects updated in the pull request description).

### Resources/Instructions
https://github.com/hackforla/website/wiki/Template-of-a-project.md-file

**Comment by ExperimentsInHonesty** [comment]
@wesrowe I think you meant to post the content you put here: https://github.com/hackforla/website/issues/459#issuecomment-623030779
into this issue.   Please delete it from issue #459 and repaste here.

**Comment by ExperimentsInHonesty** [comment]
@cnk posted this review on pull @ruben1s pull request
https://github.com/hackforla/website/pull/480#discussion_r418717610

**Comment by wesrowe** [comment]
@harishlingam @yuikomajima or potentially others — I was just looking for the best entry point (link) to give to reactJS developers.
I'll say at the outset, maybe I'm missing something, or maybe you already have this in the planning stage...
Use case: I'm trying to market onboarding nights on social media. Most design/tech communities are organized around specific flavors of tech or design -- so a social media message (eg tweet) is going to be tailored to, e.g., ux designers, react devs, javascript devs, even devops and QA – by using hashtags. I just this afternoon tweeted at (hashtags for) nodeJS and reactJS.
Linking for a link to add to the tweet, I found the info a little fragmented on the current website. I wrote a tweet-thread to give people tips on finding their way around, but the experience of finding e.g. a react project is clunky:

    homepage has "current needs" in leads' words (arguably a little hard to parse sometimes). Can be helpful if someone takes the time to read carefully.
    project cards have languages (and in the future, frameworks, right?).

Do you have a suggestion for the best place to land a skill-focused visitor? or a feature idea you could prioritize?
Thanks!

**Comment by yuikomajima** [comment]
@wesrowe If I am understanding right, I think the best link to share is a link that leads to a pre-filtered list of projects. We are in the process of creating filters on the home page and once that's done we may move on with creating a project page with the same function. The new home page may include just curated projects that are pre-filtered (open roles for backend developers, designers etc.) We just started talking about this last week so still a work in progress but if that helps answer your question!
